src/Serilog.Sinks.SumoLogic/LoggerConfigurationSumoLogicExtensions.cs

```diff
@@ -24,6 +25,7 @@ public static class LoggerConfigurationSumoLogicExtensions
         /// <param name="period">The time to wait between checking for event batches.</param>
         /// <param name="textFormatter">Supplies how logs should be formatted, or null to use the default</param>
         /// <param name="outputTemplate">Override default output template. Should not be used if overriding <see cref="ITextFormatter"/></param>
+        /// <param name="handler">Override default http handler <see cref="ITextFormatter"/></param>
```

Comment:
Change ITextFormatter to HttpMessageHandler

Reply:
Updated